pkg/metaserver/util/util.go

```diff
@@ -14,12 +16,34 @@ import (
 	"k8s.io/klog/v2"
 
 	beehiveModel "github.com/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/edge/pkg/common/client"
 )
 
 const (
 	EmptyString = ""
 )
 
+var (
+	CRDResourceToKind = make(map[string]string)
+	CRDKindToResource = make(map[string]string)
```

Comment:
I suggest you consider using k8s native methods directly.Its function is very complete.
Can refer to:
https://github.com/kubernetes/apimachinery/blob/b255da54548ac79a2ff82920dfe06598ed3f0782/pkg/api/meta/restmapper.go#L57

Reply:
I have submitted some updates, plz review!